Sources/Widgets/Representations/PolyLineRepresentation/index.js

```diff
@@ -45,6 +45,21 @@ function vtkPolyLineRepresentation(publicAPI, model) {
     return model.points;
   }
 
+  /**
+   * Change the line/tube thickness.
+   * @param {number} lineThickness
+   */
+  function applyLineThickness(lineThickness) {
+    let scaledLineThickness = lineThickness;
+    if (publicAPI.getScaleInPixels()) {
+      const center = vtkBoundingBox.getCenter(
+        model.internalPolyData.getBounds()
+      );
+      scaledLineThickness *= publicAPI.getPixelWorldHeightAtCoord(center);
```

Comment:
Does this also work if the line is not perpendicular to the viewing direction? Seems like the line would look like a perspective-projected cylinder to me.

Reply:
you are correct, it looks like a perspective-projected cylinder in that case.

I do not have anything better to suggest though.